src/cache.service.spec.ts

```diff
@@ -0,0 +1,218 @@
+/* tslint:disable:no-unused-variable */
+
+import { TestBed, inject } from '@angular/core/testing';
+
+import {
+  HttpTestingController,
+  HttpClientTestingModule
+} from '@angular/common/http/testing';
+
+import { APIService } from './api.service';
+import { ConfigurationService } from './configuration.service';
+import { AuthenticationService } from './authentication.service';
+import { CacheService } from './cache.service';
+import { Observable } from 'rxjs/Observable';
+
+const front_page_response = {
+      "@id": "http://fake/Plone/",
+      "@type": "Plone Site",
+      "id": "Plone",
+      "items": [
+        {
+          "@id": "http://fake/Plone/front-page",
+          "@type": "Document",
+          "description": "Congratulations! You have successfully installed Plone.",
+          "title": "Welcome to Plone"
+        }
+      ],
+      "items_total": 5,
```

Comment:
Just curious:  why is `items_total` 5?

Reply:
the Plone RESTAPI always paginates items, so you get only the 20 first ones, but you can now how many there are with items_total